### PR TITLE
CVE-2015-1266 and CVE-2015-6788

### PR DESCRIPTION
**Pull request by devinmatte**

### CODE REVIEW

File: cves/CVE-2015-1266.yml [hunk 1]
```diff
@@ -32,7 +32,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Attackers were able to bypass restrictions in chrome to access GUI components
+  that should have only been accessable with certain permissions or through a proper
```
Comment (jacobklssn) [inline cves/CVE-2015-1266.yml]: 'accessable' should be 'accessible'.

File: cves/CVE-2015-6788.yml [hunk 2]
```diff
@@ -32,7 +32,8 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Some handler functions were improperly implemented. This allowed for attackers to cause a denial of service or utilize potential type confusion.
```
Comment (jacobklssn) [inline cves/CVE-2015-6788.yml]: A quick description of how the handler functions were improperly implemented would make this description sound better. At this point, it sounds like ,"something was done wrong, and people could take advantage of it." which doesn't really describe the vulnerability.

File: cves/CVE-2015-6788.yml [hunk 3]
```diff
@@ -119,7 +122,9 @@ interesting_commits:
     emerging themes?
 
     If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    There were not really any interesting commits. Between the initial code and the fix there were several years where the code stayed entirely untouched.
+    The fix came about 2 years later with the lines remaining untouched until then.
```
Comment (jacobklssn) [inline cves/CVE-2015-6788.yml]: Was this vulnerability not found for a very long time, or did it not have a high priority and was just not fixed for a long time?

File: cves/CVE-2015-6788.yml [hunk 4]
```diff
@@ -155,37 +160,39 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies: 
+    applies: false 
     note:
```
Comment (jacobklssn) [inline cves/CVE-2015-6788.yml]: You talk below about problems that can work their way up the chain of a program through trusted input. Could this be a good example of defense_in_depth, as the program could have been prepared in some way for the misuse of trusted input?

File: cves/CVE-2015-6788.yml [hunk 5]
```diff
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
```
Comment (jacobklssn) [inline cves/CVE-2015-6788.yml]: upvotes: 2

File: cves/CVE-2015-1266.yml [hunk 6]
```diff
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
```
Comment (jacobklssn) [inline cves/CVE-2015-1266.yml]: upvotes: 4

File: cves/CVE-2015-1266.yml [hunk 7]
```diff
@@ -32,7 +32,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Attackers were able to bypass restrictions in chrome to access GUI components
+  that should have only been accessible with certain permissions or through a proper
+  route.
```
Comment (tjb4578) [inline cves/CVE-2015-1266.yml]: How were attackers able to bypass restrictions?  Looking at your 'lessons' section, it looks like attackers were able to provide unanticipated input in the URL bar?

File: cves/CVE-2015-1266.yml [hunk 8]
```diff
@@ -55,16 +58,16 @@ fixes:
...
+  :note: ''
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 4
```
Comment (tjb4578) [inline cves/CVE-2015-1266.yml]: +4

File: cves/CVE-2015-6788.yml [hunk 9]
```diff
@@ -65,7 +66,7 @@ upvotes_instructions: |
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 2
```
Comment (tjb4578) [inline cves/CVE-2015-6788.yml]: +1

**Pull request merged**